src/pybind/mgr/mgr_module.py

```diff
@@ -665,6 +707,32 @@ def notify(self, notify_type, notify_id):
         """
         pass
 
+    def _config_notify(self):
+        # check logging options for changes
+        current_level = logging.getLevelName(self._ceph_log_handler.level)
+        level = self.get_module_option("log_level")
+        if level.lower() not in ['info', 'debug', 'critical', 'error', 'warning', 'notset']:
```

Comment:
this can be enforced by the mon if you define enum_allowed for the module option above